src/index.js

```diff
@@ -1,21 +1,48 @@
-import _ from 'lodash';
-import './style.css';
-import Icon from './icon.png';
+class Task {
+  static tasks = [
+    {
+      id: 0,
+      description: 'Coding',
+      completed: false,
+    },
+    {
+      id: 1,
+      description: 'Play game',
+      completed: false,
+    },
+    {
+      id: 2,
+      description: 'Watch a movie',
+      completed: true,
+    },
+  ];
 
-function component() {
-  const element = document.createElement('div');
+  constructor(description) {
+    this.id = Task.tasks.length;
+    this.description = description;
+    this.completed = false;
+  }
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-
- // Add the image to our existing div.
- const myIcon = new Image();
- myIcon.src = Icon;
-
- element.appendChild(myIcon);
-
-  return element;
+  static toggleTaskStatus(id) {
+    Task.tasks.forEach((task) => {
+      if (task.id === id) {
+        task.completed = !task.completed;
+      }
+    });
+  }
 }
-
-document.body.appendChild(component());
\ No newline at end of file
+window.onload = () => {
+  const todoList = document.querySelector('.todo-list');
+  Task.tasks.forEach((task) => {
```

Comment:
- [optional] On page load, the task array is rendered dynamically to the DOM and the tasks should appear in order of the index values for each task. **Please kindly implement this functionality.**